test/addurl_test.go

```diff
@@ -69,8 +69,10 @@ TAG ` + tag
 	})
 	defer os.RemoveAll(buildDir)
 
-	err := runRockerBuildWdWithOptions(buildDir, "-cache-dir", cacheDir)
-	assert.Nil(t, err, "no difference in downloaded and added files")
+	err := runRockerBuildWd(buildDir, "-cache-dir", cacheDir)
+	if err != nil {
```

Comment:
why you don't use 
```go
if assert.Nil(t, err) 
...
``` 
? I think it will be better if we try to use testify asserting style. https://godoc.org/github.com/stretchr/testify/assert#Nil